lib/common/attrs.c

```diff
@@ -79,9 +79,9 @@ pcmk__node_attr_target(const char *name)
 char *
 pcmk_promotion_score_name(const char *rsc_id)
 {
-    if (rsc_id == NULL) {
+    if (rsc_id == NULL || strlen(rsc_id) == 0) {
```

Comment:
we have a convenience function for this, pcmk__str_empty(rsc_id)

Reply:
Ah, right :)